magenta-lib/src/main/scala/magenta/deployment_type/AmiCloudFormationParameter.scala

```diff
@@ -33,6 +33,11 @@ object AmiCloudFormationParameter extends DeploymentType {
   val amiParameter = Param[String]("amiParameter",
     documentation = "The CloudFormation parameter name for the AMI"
   ).default("AMI")
+  val amiParametersToTags = Param[Map[String, Map[String, String]]]("amiParametersToTags",
```

Comment:
I didn't find it very easy at first to understand what all those strings represent. Would it make sense to create types to help readability and, more importantly, to restrict what it can contain?